geode-core/src/main/java/org/apache/geode/internal/net/SSLParameterExtension.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.internal.net;
+
+import java.util.Properties;
+
+import javax.net.ssl.SSLParameters;
+
+/**
+ * User implementation of a SSLParameter extension logic.
+ *
+ * @since Geode 1.12
+ */
+
+public interface SSLParameterExtension {
+  /**
+   * Initialize the SSLParameterExtension.
+   *
+   * @param properties the properties
+   */
+  default void init(Properties properties) {}
```

Comment:
I think the initialization should receive a `DistributionConfig` object. That will also be much clearer to implementors of this interface as to what is being passed in.

Reply:
Thanks for comments